### ISSUE TITLE
Bigger ship shadow

### ISSUE THREAD

**Issue by harry-wood** [matthias-schuetz/webgl-platforms-game#1]
I think it would help with 3D perception of the position of the ship, and whether you're landing ok on the platform or not...    if the shadow of the ship was bigger, and preferably it would change size depending on how high above the platform it is.

At the moment I see a small dot vertically below the ship, which I guess is the shadow being rendered, but it's just an unchanging very small dot.

**Comment by matthias-schuetz** [comment]
Yes, the small dot was added on purpose: I didn't like the appearance of the default THREE.js/WebGL shadows since they're a bit blurry. I tried to project a custom made shadow on the platforms which represented the shape of the ship. The problem is that such a shadow shape doesn't adjust and occlude correctly on those small light-green corner platforms. You would see that it's not a real shadow since the shape would kind of float on the light-green platforms. That's why I decided to go with a simple dot.
Maybe you've got an idea how to solve the problem of a crisp shadow that behaves correctly on stacked objects and slants. I couldn't come up with that so far.
